pages/Mining.qml

```diff
@@ -106,27 +106,59 @@ Rectangle {
                 Layout.fillWidth: true
                 spacing: 16
 
-                MoneroComponents.LineEdit {
-                    id: soloMinerThreadsLine
-                    Layout.minimumWidth: 200
-                    text: "1"
-                    validator: IntValidator { bottom: 1; top: idealThreadCount }
-                }
+                RowLayout {
+                    MoneroComponents.StandardButton {
+                        id: removeThreadButton
+                        small: true
+                        primary: false
+                        text: qsTr("-") + translationManager.emptyString
+                        enabled: soloMinerThreadsLine.text > 1
+                        onClicked: {
+                                soloMinerThreadsLine.text = Number(soloMinerThreadsLine.text) - 1
+                        }
+                    }
 
-                MoneroComponents.TextPlain {
-                    id: numAvailableThreadsText
-                    text: qsTr("Max # of CPU threads available for mining: ") + idealThreadCount + translationManager.emptyString
-                    wrapMode: Text.WordWrap
-                    font.family: MoneroComponents.Style.fontRegular.name
-                    font.pixelSize: 14
-                    color: MoneroComponents.Style.defaultFontColor
+                    MoneroComponents.TextPlain {
+                        id: soloMinerThreadsLine
+                        Layout.bottomMargin: 1
+                        Layout.minimumWidth: 45
+                        color: MoneroComponents.Style.defaultFontColor
+                        text: idealThreadCount
```

Comment:
Setting all available threads as default isn't a good idea, I would undo it to 1, like it was previously.

Reply:
oops my mistake, I intended to set it to recommended value (idealThreadCount / 2). Don't you think it's better than 1?

Reply:
I think previously it was at 1. `idealThreadCount / 2` would work too.